### ISSUE TITLE
There is no translation head in the code？

### ISSUE THREAD

**Issue by EasonEdison** [LZGMatrix/BOP19_CDPN_2019ICCV#3]
Hello, Mr li, I can`t find the translation head in resnet18, and T_vercor is coming from PnP, Why isn't the highlight of your paper in the code?

**Comment by LZGMatrix** [comment]
This code is for the BOP2019 challenge. In this challenge, the biggest difficulty is the domain adaption between the synthetic training data and real test data, which is our main focus.

**Comment by EasonEdison** [comment]
So that's it. So will you update the relevant parts of the paper?

**Comment by flowtcw** [comment]
I am also looking forward to the original code for ICCV2019.

**Comment by EasonEdison** [comment]
> I am also looking forward to the original code for ICCV2019.

作者这么久没回复，可能是没戏了 0. 0

**Comment by flowtcw** [comment]
> > I am also looking forward to the original code for ICCV2019.
> 
> 作者这么久没回复，可能是没戏了 0. 0

心怀希望，233

**Comment by LZGMatrix** [comment]
> I am also looking forward to the original code for ICCV2019.

The original code for ICCV2019 will be provided in the near future, please be patient ~

**Comment by EasonEdison** [comment]
> > I am also looking forward to the original code for ICCV2019.
> 
> The original code for ICCV2019 will be provided in the near future, please be patient ~
Could you teach me how to calculate the loss function of rotation, which is not mentioned in the paper?

**Comment by zhangxiaodi** [comment]
没有源码确实遗憾，有点为难刚入门的萌新了！
